trello.py

```diff
@@ -0,0 +1,90 @@
+import os
+import pprint
+import requests
+
+pp = pprint.PrettyPrinter(indent=4)
+app_board_name='To Do App'
+trello_member='brendanmcdaniel'
+trello_endpoint='https://api.trello.com/1/'
+
+class item:
+    def __init__(self, id, status, title):
+        self.id = id
+        self.status = status
+        self.title = title
+
+class status:
+    def __init__(self, id, name):
+        self.id = id
+        self.name = name
+
+# Check if proxy env vars have been set
+try:
+    http_proxy = os.environ['http_proxy']
+except KeyError:
+    pp.pprint('No HTTP proxy found')
+    http_proxy = ''
+try:
+    trello_key = os.environ['TRELLO_KEY']
+    trello_token = os.environ['TRELLO_TOKEN']
+except KeyError:
+    pp.pprint('Trello env vars dont seem to be set')
+    trello_key = ''
+    trello_token = ''
+
+# Environment variables set for trell api
```

Comment:
Example of comments being a pain to update and ending up misleading. I assume this used to precede the trello_key/trello_token assignments. If you find it is confusing without comments, then consider refactoring it instead of adding a comment.